CREATE DATABASE  IF NOT EXISTS `db_bacteria2` /*!40100 DEFAULT CHARACTER SET utf8 */;
USE `db_bacteria2`;
-- MySQL dump 10.13  Distrib 5.6.11, for Win32 (x86)
--
-- Host: 127.0.0.1    Database: db_bacteria2
-- ------------------------------------------------------
-- Server version	5.5.0-m2-community

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `basicinfo`
--

DROP TABLE IF EXISTS `basicinfo`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `basicinfo` (
  `Ecoid` varchar(9) NOT NULL,
  `evidencelevel` int(11) NOT NULL,
  `projectname` varchar(200) NOT NULL,
  `contributor` varchar(45) DEFAULT NULL,
  `Email` varchar(60) DEFAULT NULL,
  `description` text NOT NULL,
  `reference` varchar(120) NOT NULL,
  `author` varchar(200) NOT NULL,
  `year` varchar(4) NOT NULL,
  `confirmation` tinytext,
  `prodiagram` varchar(80) DEFAULT NULL,
  `cctdescription` text,
  `cctdiagram` varchar(80) DEFAULT NULL,
  PRIMARY KEY (`Ecoid`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `basicinfo`
--

LOCK TABLES `basicinfo` WRITE;
/*!40000 ALTER TABLE `basicinfo` DISABLE KEYS */;
INSERT INTO `basicinfo` VALUES ('Eco000_09',3,'ECO-FUELE.COLINE','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','this project aims to increase the production of isobutanol.','http://2012.igem.org/Team:NCTU_Formosa','NCTU Formosa','2012','1',NULL,NULL,NULL),('Eco000_10',3,'Pepdex','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to quick deliver peptides of specific therapeutic use.This circuit was named Fatty Acid Inducible.','http://2012.igem.org/Team:NTU-Taida','NTU-Taida','2012','0',NULL,NULL,NULL),('Eco000_11',3,'Pepdex','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to quick deliver peptides of specific therapeutic use.','http://2012.igem.org/Team:NTU-Taida','NTU-Taida','2012','0',NULL,NULL,NULL),('Eco000_12',3,'Biowave','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to utiliz a biological system with the properties of negative feedback and time delay to perform a synchronized oscillation.And they use this system to simulate e.g.traffic jams between cities.','http://2012.igem.org/Team:Fudan_Lux','Fudan_Lux','2012','The result is a perfect match for the simulate.And they have constructe the modeling.',NULL,NULL,NULL),('Eco000_13',3,'Bio-capsule E.coli','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make bio-capsules formed by aggregation of E. coli through cell-cell interaction via adhesion molecule, Ag43, located on the surface of outer membrane.This circuit is a process about producing P(3HB).','http://2012.igem.org/Team:HokkaidoU_Japan','HokkaidoU_Japan','2012','1',NULL,NULL,NULL),('Eco000_14',3,'Bio-capsule E.coli','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make bio-capsules formed by aggregation of E. coli through cell-cell interaction via adhesion molecule, Ag43, located on the surface of outer membrane.This circuit is aggregation and characterization module.','http://2012.igem.org/Team:HokkaidoU_Japan','HokkaidoU_Japan','2012','1',NULL,NULL,NULL),('Eco000_15',3,'Programming Cells Through Light','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to utiliz an ultra-sensitive engineered sensor of luminescence -- the?Luminesensor.With this sensor, they would be able to detect natural light and even bioluminescence,and spatiotemporal control of cellular behavior,and implemente cell-cell signaling using light for very first time .','http://2012.igem.org/Team:Peking','Peking','2012','1',NULL,NULL,NULL),('Eco000_16',3,'Breaking the Symmetry','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make the bacteria grow into the form of a star.They try to design a biobrick system which let the bacteria take the irregular shape all by itself, despite external conditions.','http://2012.igem.org/Team:SEU_O_China','SEU_O_China','2012','They designed the modeling. They have tested the light sensors and have proved FtsZ truly represses the growth of bacteria.But they didn\'t finish the experience.',NULL,NULL,NULL),('Eco000_17',3,'Domino Effect in E.coli','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to establish a novel information processing system in bacterial community, based on the logic of domino effect and quorum sensing system, which possesses the capability of information transmission, amplification and logic gating.','http://2012.igem.org/Team:Tsinghua','Tsinghua','2012','They divided the project into three parts, and do some detection. They didn\'t finish the experience, we didn\'t see the result.',NULL,NULL,NULL),('Eco000_18',3,'Romeo and Juliet','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to generate a circuit involving regulatory mechanism of positive feedback rather than commonly-used negative feedback to control the fate of E.coli by signaling between two types of E.coli.And they recreate the story of \\\"Romeo an Juliet\\\"','http://2012.igem.org/Team:Tokyo_Tech','Tokyo_Tech ','2012','1',NULL,NULL,NULL),('Eco000_19',3,'anti-phage E.coli','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to design a gene circuit which can both sense and defend against phages.','http://2012.igem.org/Team:USTC-China','USTC-China','2012','They test the final system.',NULL,NULL,NULL),('Eco000_20',3,'E.lumoli','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to constructe a fluorescent digital display device with synthetic logic gates,which is able to respond to signals by displaying and switching numbers.','http://2012.igem.org/Team:XMU-China','XMU-China','2012','They have constructed and characterized a fluorescent digital display device with synthetic gene circuits,which is able to respond to signals by displaying and switching number 0 and 1.',NULL,NULL,NULL),('Eco000_21',3,'plastic terminator','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make a synthetic bacteria that can degrade plastic especially PE/PET with a novel degrading enzymes.','http://2012.igem.org/Team:BAU-Indonesia','BAU-Indonesia','2012','0',NULL,NULL,NULL),('Eco000_22',3,'degrade oil','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to degrade oil, and solve the problem of oil pollution.','http://2012.igem.org/Team:CD-SCU-CHINA','CD-SCU-CHINA','2012','0',NULL,NULL,NULL),('Eco000_23',3,'S.aureus monitor','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to constructe an E.coli biofilm consisting of two different engineered polutions, which are designed to detect and eradicate S.aureus.This circuit is Biokiller','http://2012.igem.org/Team:HIT-Harbin','HIT-Harbin','2012','They choose lysostaphin as a new \\\"tool\\\" to kill S.aureus. And they detect some parts.',NULL,NULL,NULL),('Eco000_24',3,'S.aureus monitor','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to constructe an E.coli biofilm consisting of two different engineered polutions, which are designed to detect and eradicate S.aureus.This circuit is Biosensor.','http://2012.igem.org/Team:HIT-Harbin','HIT-Harbin','2012','?The results confirmed that the detecting device cen be able to sense AIPs natively produced byS.aureus',NULL,NULL,NULL),('Eco000_25',3,'Oceanfloat and Oceanfeel','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to construct various toolkits for red tide prevention and management.','http://2012.igem.org/Team:OUC-China','OUC-China','2012','They divided the project into three parts, and do some detection. They didn\'t finish the experience, we didn\'t see the result.',NULL,NULL,NULL),('Eco000_26',3,'Conquer venus','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to the removal of several pollutants, including nitrogen oxides, sulfur oxides and carbon oxides, from exhaust air, waste water and cadmium ions from soil. Then human beings can colonize Venus and expand our territory.','http://2012.igem.org/Team:NYMU-Taipei/ymiproject.h','NYMU-Taipei','2012','They didn\'t finish their works.',NULL,NULL,NULL),('Eco000_27',3,'MI','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to built several genetic circuits that work sequentially to repair the heart.Then they hope it can work in cell culture models of myocardial infarction(MI).','http://2008.igem.org/Team:Bay_Area_RSI','Bay_Area_RSI','2008','We have shown that cells bearing the CRP receptor system specifically bind to CRP deposited on damaged cardiomyocytes, and that the signal is then relayed to turn on expression of downstream effector genes.But they didn\'t finish their works.',NULL,NULL,NULL),('Eco000_28',3,'Toxipop','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to create a biosensor that interfaces biological and electrical systems and acts as a novel electrical reporting system, for which one use would be to detect toxins in water supplies.','http://2008.igem.org/Team:Brown','Brown','2008','0',NULL,NULL,NULL),('Eco000_29',3,'Curing Lactose Intolerance','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to engineer a member of this microbial community to provide innovative medical treatments.This bacteria is about Lactose Intolerance','http://2008.igem.org/Team:Caltech','Caltech','2008','They did the first plasmid--Lactose Regulator ,but they didn\'t finish it . And they didn\'t construct the second plasmid.',NULL,NULL,NULL),('Eco000_30',3,'E.nigma','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to work toward the design and construction of a bacterial hash function.','http://2008.igem.org/Team:Davidson-Missouri_Wester','Davidson-Missouri Western','2008','They design the modeling.',NULL,NULL,NULL),('Eco000_31',3,'MicroMaize','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make an engineering bacteria  to produce starch from the cellulose in waste biomass. ','http://2008.igem.org/Team:Edinburgh','Edinburgh','2008','They test many parts,and the results are promising but their dream is still unrealized.',NULL,NULL,NULL),('Eco000_32',3,'MicroMaize','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make an engineering bacteria  to produce the vitamin A precursor beta-carotene. ','http://2008.igem.org/Team:Edinburgh','Edinburgh','2008','They encountered problem at final step.',NULL,NULL,NULL),('Eco000_33',3,'MicroMaize','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make MicroMaize more appealling, they would make it smell and taste of lemons.','http://2008.igem.org/Team:Edinburgh','Edinburgh','2008','The parts still being tested.They can\'t be sure whether the limonene synthase is working in their host.',NULL,NULL,NULL),('Eco000_34',3,'GM endosymbionts','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Reprogramming microbes to cater to or silence their hosts - beta carotene?production and RNAi delivery. They are interested in taking advantage of some of these microbes to deliver transgenic payloads for the benefit or modification of the host organism.These might be called GM endosymbionts.','http://2008.igem.org/Team:Guelph','Guelph','2008','They didn\'t finish the beta carotene production.',NULL,NULL,NULL),('Eco000_35',3,'bactricity','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Our project sought to combine the detecting capabilities of bacteria with the speed and ubiquity of electricity by creating an inducible system in Shewanella oneidensis MR-1 with an electrical output, allowing for the direct integration of this biosensor with electrical circuits via microbial fuel cells.','http://2008.igem.org/Team:Harvard','Harvard','2008','They have done some tests,and these results are exciting in that they show a possibility of taking advantage of this cooperative effort to achieve inducible current.',NULL,NULL,NULL),('Eco000_36',3,'Dr. Coli','SUSTC-ShenzhenB',';igem2013@sustc.edu.cn','This project aims to construct an E. coli bacterium that produces a drug when and where it is needed in the human body.','http://2008.igem.org/Team:KULeuven','KULeuven','2008','They divide the project into 7 parts: input,output, filter, invertimer, reset, cell death and memory.Then they construct the plasimds,and testing some parts.',NULL,NULL,NULL),('Eco000_37',3,'Singing bacteria','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aims to make bacteria sing, and it will be done by modifying the extracellular medium\'s resistivity through the modulation of the RcnA (E. coli\'s natural efflux pump) which will in turn change the concentration of nickel outside the cell.','http://2008.igem.org/Team:LCG-UNAM-Mexico','LCG-UNAM-Mexico','2008','They carry out the experiments,measurements, and controls to analyze the results.The result is shown that resistivity can be readily related to nickel concentration even in the presence of living cells.',NULL,NULL,NULL),('Eco000_58',3,'Targeted Bacterial Therapeutics','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','this engineering E.coli displays an antibody-mimetic protein on the surface of E.coli.This protein has a high affinity to HER2 which can effectiviely recognize and adhere to breast cancer tumors.With blue light stimulation,this engineering E.coli secretes cytolysin A to kill breast cancer.','http://2012.igem.org/Team:Penn/ProjectOverview','University of Penn','2012','1',NULL,NULL,NULL),('Eco200_00',3,'E.coli Time Manager','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','The transcription factors(TFs) in the \\\"timer\\\" cells can be activated by a certain amount of inducer(AHL).When the TFs switches on,the reporter(GFP) will tell the time.','http://2009.igem.org/Team:Chiba','Chiba','2009','1',NULL,NULL,NULL),('Eco200_04',3,'AI-2 Signalling System','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Vibrio harveyi AI-2 signalling system in E.coli','http://2009.igem.org/Team:Calgary','Calgary','2009','0',NULL,NULL,NULL),('Eco200_06',3,'E.colight','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Using fusion proteins to regulate gene expression through light control.','http://2009.igem.org/Team:EPF-Lausanne','EPF-Lausanne','2009','1',NULL,NULL,NULL),('Eco200_07',3,'Low suger stheyetener producer','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Using synthetic biology to engineer pro-biotic lactic acid bacteria used in the production of dairy products like yogurt, buttermilk and curds, to produce Monellin, a heat and pH stable stheyetening protein.','http://2010.igem.org/Team:IIT_Madras','IIT_Madras','2010','0',NULL,NULL,NULL),('Eco200_09',3,'E.coli Pen','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Using E.coli as a color stheirce to create the three primary colors.','http://2010.igem.org/Team:KIT-Kyoto','KIT-Kyoto','2010','1',NULL,NULL,NULL),('Eco200_11',3,'Bioclock','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Using oscillator theory to create an E.coli that can tell the time by expressing Fluorescent Protein.','http://2009.igem.org/Team:Todai-Tokyo','Todai-Tokyo','2009','0',NULL,NULL,NULL),('Eco200_13',3,'Bread','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Using yeast to make sweet bread without adding any sugar in order to prevent people from absorbing too much suger.','http://2009.igem.org/Team:Todai-Tokyo','Todai-Tokyo','2009','0',NULL,NULL,NULL),('Eco200_14',3,'Arterisclerosis','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Reducing LDL by introducing LDL receptor to yeast. LDL receptor catches LDL therefore reduces LDL concentration.','http://2009.igem.org/Team:Todai-Tokyo','Todai-Tokyo','2009','0',NULL,NULL,NULL),('Eco200_16',3,'Bactomotor','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','\\\"Bactomotor\\\" aims to overcome the general problems that nanorobots lack a reliable propulsion poTheyr source to operateto bring about a foundational and pioneering advancement in the field.','http://2009.igem.org/Team:HKU-HKBU','HKU-HKBU','2009','0',NULL,NULL,NULL),('Eco200_17',3,'Lactoguard','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','The chassis of our machine, Lactobacillus gasseri, is a predominant commensal vaginal microbe and is ideally situated to detect and deter Human Papillomavirus(HPV).\nInterbacterial communication is achieved by quorum sensing and each bacterium will produce a visible reporter on receipt of the signal which the infected woman would be able to detect without medical assistance shortly after infection.','http://2010.igem.org/Team:WITS-South_Africa','WITS-South_Africa','2010','0',NULL,NULL,NULL),('Eco200_19',3,'E.coli Traffic Light','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','\\\"E.coli Traffic Light\\\" aims to create a biosensor that recognizes a specific target and alters its output fluorescence from green, to yellow, to red as a function of concentration up to critical levels (hence, a biological \\\"traffic light\\\").','http://2009.igem.org/Team:British_Columbia','British_Columbia','2009','0',NULL,NULL,NULL),('Eco200_20',3,'Pico Plumber','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Pico Plumber can swim towards to the leaking site where aspartate is released and produce the glue only when close to the breach. By the quorum sensing signal, bacteria \\\"know\\\" that they have reached the leaking site, and therefore, they can start producing the glue. The bacteria lyse after a certain time interval and release the glue protein to repaire the hole.','http://2009.igem.org/Team:Aberdeen_Scotland','Aberdeen_Scotland','2009','0',NULL,NULL,NULL),('Eco400_00',3,'genetic double click generator vission1','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Constructe genetic double-click system whose output is released only when the input (inducing agent) is given twice within a limited time. This mechanism could work as a sort of safety device.','http://2010.igem.org/Team:Chiba','Chiba University','2010','1',NULL,NULL,NULL),('Eco400_01',3,'genetic double click generator vission2','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Constructe genetic double-click system whose output is released only when the input (inducing agent) is given twice within a limited time. This mechanism could work as a sort of safety device.','http://2010.igem.org/Team:Chiba','Chiba University','2010','1',NULL,NULL,NULL),('Eco400_02',3,'Bio-clock','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Build a biological clock that can count up when it is \\\"ticked\\\" by input light pulse signals. As the signal \\\"ticks\\\" the clock, the clock changes color and observer can use the color or combination of colors to tell what \\\"time\\\" it is. ','http://2008.igem.org/Team:Melbourne','Melbourne','2008','This team has designed the circuits for their clock, but did not succeed.\nParts Designed\nlacI promoter + Key3C\npTet->lock3d->GFP->Ter\nParts Constructed\nlacI promoter + Key3C\n',NULL,NULL,NULL),('Eco400_03',3,'Light controlled heavy metal carrying E. coli','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Design a E.coli machine to control heavy metal concentration in water by using phototaxis induced tunable carrying system on E.coli membrane. There are two light sources in this system. One of the two is for the metal source and another one is for the destination to where we want to carry metal atoms. In addition, they have metal binding domain in outer surface and bacteriorhodopsin. When we open light at the metal source, they start to move toward metal source and catch the metal atoms. Then we open the light at the destination while closing the light at the source. So our machines move toward the destination. After the arrival our machines will be exposed to light with specific wavelength to activate bacteriorhodopsin leading to small local pH changing at the outer surface. This pH changing causes the release of metal atoms from the metal binding domain. Eventually, system starts from the beginning and so on.','http://2008.igem.org/Team:METU_Turkey','METU Turkey','2008','They didn\'t mention in their wiki',NULL,NULL,NULL),('Eco400_04',3,'horizontal gene transfer','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Design a devise that would detect events of horizontal gene transfer among bacteria. In Prokaryotes there is three ways in which HGT can occur. \nThey designed a couple plasmids so that they confer different antibiotic resistance and express a reporter gene when they are in the same cell. Namely, they grow a bacteria that has one of the plasmids in a media containing a sublethal dosis of antibiotic, which the bacteria lacks its resistance. Then they add the complementary plasmid to the media. So they will be able to detect HGT events on basis of the bacterial\ngrowth and the expression of the reporter gene.','http://2008.igem.org/Team:Mexico-UNAM-IPN','Mexico-UNAM-IPN','2008','0',NULL,NULL,NULL),('Eco400_05',3,'lignin dregadation','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Lignin is very difficult to degrade£¬and it lends the ¡®hardness¡¯ to wood materials. Its limitation on wood product bioavailability is due to the complex polymeric structure which proves very difficult to break. Therefore , it is hard to utilization of biomass resource.\nThis project build parts to degrade lignin. In addition, they want to build standard parts for biodegradation. ','http://2008.igem.org/Team:Mississippi_State','Mississippi State','2008','Isolated Lignin Peroxidase(BBa_144000):\n1.1117 bp Lignin Peroxidase Isozyme\n2.Currently under testing and characterization\n3.Undergoes redox reaction with Veratryl Alcohol and Hydrogen Peroxide substrates\n4.Used to degrade Lignin and possibly other comple',NULL,NULL,NULL),('Eco400_06',3,'alcohol sensor','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','The alcohol sensor is a device created from living microorganisms.The alcohol sensor is based on the metabolic activity of the yeast Pichia pastoris. Through genetic modification, these cells may indicate the concentration of alcohol present in their environment.\nMethanol concentration will correspond to the time or intensity of fluorescence.  Ethanol concentration may be determined from the length of time it takes for the cells to produce a visible signal after both ethanol and methanol have been introduced to the system.  This signal mechanism is created through the fusion of the alcohol oxidase gene (AOXI) promoter and a red fluorescent protein indicator in a vector.  ','http://2008.igem.org/Team:Missouri_Miners','Missouri Miners','2008','The alcohol sensor is still in the construction phase. They build a part BBa_I764001 - AOX1.\nBesides, there are some problems in this project:\nmethanol has to be present for ethanol concentration measuring.\n2.the lag time before the red fluorescence can b',NULL,NULL,NULL),('Eco400_07',3,'Bio-gurt','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Engineer yogurt bacteria to prevent streptococcus mutants from binding to the tooth,while streptococcus mutants is a leading cause of dental caries.','http://2008.igem.org/Team:MIT','MIT','2008','1.Designed an expression system for protein secretion in Lactobacillus.\n2.Successfully transformed Lactobacillus bulgaricus and Lactobacillus lactis.\n3.Successfully developed a novel binding assay to evaluate p1025.\nP1025(k128003): found to competitively ',NULL,NULL,NULL),('Eco400_08',3,'A Genetic Circuit for Directed Evolution in v','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Directed evolution is a powerful tool for answering scientific questions or constructing novel biological systems. This project present a simple genetic circuit for in vivo directed evolution which comprises minimal elements for random mutation and artificial selection. They engineer yeast to generate the DNA mutator hAID, an essential protein in adaptive immunity, and target it specifically to a gene of interest. The target gene will be mutated and consequently promptly evolves. By linking the expression of hAID repressor LacI and favorite gene functionality, the mutation rate inversely correlates between the functionality of the desired gene and hAID.','http://2008.igem.org/Team:Peking_University','Peking university','2008','The result of their experiments were not satisfactory, but they had build many parts, which may be useful for you:\nK120000~k120024',NULL,NULL,NULL),('Eco400_09',3,'Diauxie elimination by xylose inducible promo','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Microorganisms typically prefer to utilize glucose over other sugar carbon sources such as xylose.  In E. coli, the xylose metabolism operon is controlled by both the xylose-inducible XylR activator protein and the cAMP receptor protein (CRP). In this project they attempt to eliminate glucose control over xylose-inducible gene expression in E. coli by altering the natural transcriptional control region of the xylose operon. Designs constructed and tested include scrambling the CRP binding site, increasing the strength of the xyl promoter, and overexpressing XylR. Xylose-inducible gene expression that functions independently of glucose regulation provides a useful approach to improving microbial utilization of biomass feedstocks containing mixtures of glucose and xylose.','http://2008.igem.org/Team:PennState','Penn State','2008','They didn¡¯t find the conditions where glucose/xylose diauxie is eliminated.\n One of their hypotheses is that XylR interacts with RNA polymerase to strengthen its binding. Xylose-bound XylR should positively regulate transcription, and the then a strength',NULL,NULL,NULL),('Eco400_10',3,'The Bio-Safety Net','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Bacteria can be designed to perform promising tasks, such as the biodegradation of oil to clean up oil spills. There are risks associated with the possibility that living bacteria may perform undesired activities. This team want to introduce a bio-safety net that will apply to virtually all genetically engineered bacteria as a vital termination step after they have finished fulfilling their function. They make this possible by introducing a \\\"suicide\\\" mechanism that is triggered under specific conditions. By using different promoters, the system can respond to changes in environmental factors and control expression specific to a chosen factor. Such mechanism can be easily assembled and incorporated into bacteria through the use of biobricks.','http://2010.igem.org/Team:HKU-Hong_Kong','HKU-Hong Kong','2010','It works well but still exist some possible flaws:\n1.Leaky expression: It might lead to the production of cl proteins which inhibit the expression of the suicide gene. This means that bacteria might still survive after they have finished their function (o',NULL,NULL,NULL),('Eco400_11',3,'Biosensor for the detection of cations','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','They are aimed at developing a biosensor for the detection of cations by introducing electron network system proteins into the chassy best suited to receive an enhanced energy flow. By facilitating the flow of electrons in a cell, we have showed an increase of ATP production. Bacterial cells use ATP for anabolic reactions, including the synthesis of DNA. Through our electron network system, we can enhance the expression of DNA, increasing cell replication along with the fluorescence of our reporters. Then these incorporated proteins can enhance the capacity of our sensor device.','http://2008.igem.org/Team:Prairie_View','Prairie View','2008','They can now test with each metal alone and in combinations with and without Cytochrome C & Ion Channels.To expand on this research, they will design their own proteins; new ion channels and cytochrome complexes that are specific to the electrical configu',NULL,NULL,NULL),('Eco400_12',3,'UV radiation biosensor','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project is creating a biosensor for UV radiation. Essentially, they are combining the SOS pathway promoter with a lacZ gene. This will lead to transcription of lacZ whenever the bacterial DNA undergoes significant damage. By plating these bacteria on X-gal plates, a blue color will be an indicator of intense UV irradiation.','http://2008.igem.org/Team:Purdue/Project','Purdue','2008','They made the part, and sent it in! Unfortunately, due to lack of funding and time, they were unable to test their part. For the expected results, you can click on the link below£ºhttp://2008.igem.org/Team:Purdue/Modeling',NULL,NULL,NULL),('Eco400_13',3,'Bio-beer','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Resveratrol has been implicated as a natural product that extends lifespan and prevents cancer, coronary disease and neurodegenerative maladies. Unfortunately, resveratrol is only present at appreciable levels in a small number of foods, such as red wine, peanuts, and blueberries. To create an alternative source for resveratrol consumption, thty are introducing a biosynthetic pathway for this compound into a brewing strain of yeast and examining whether this strain can be engineered to produce resveratrol during beer fermentation. This engineering approach should be useful for cheap biosynthesis of other oxygen-sensitive prophylactics.','http://2008.igem.org/Team:Rice_University','Rice','2008','Their cellular brewing chassis has been use in St. Arnold\'s hefeweizen beer£¬ a top-fermenting yeast which causes a great deal of gas induced convection during primary brewing. Additionally, hefeweizen beers are commonly served unfiltered, allowing for th',NULL,NULL,NULL),('Eco400_14',3,'Coli Touch','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','¡°Coli Touch¡± has a pressure sensitive display composed of an E. coli lawn. When you touch its display, touched section is colored. Its work system : Display of ¡°Coli Touch¡± contains many E. coli. When you touch this display, pressure applies to E. coli in this display. Pressure applied E. coli expresses GFP.  Compared with other kinds of input, pressure can induce uniformly.','http://2008.igem.org/Team:Tokyo_Tech','Tokyo Tech','2008','The touch display successfully regulated GFP expression in E. coli ! \nPressure induction: they have constructed pressure-inducible genetic circuit and measured its pressure response.\nTouch display: they have created the prototype - touch display. \nLow pre',NULL,NULL,NULL),('Eco400_15',3,'Bio-thermometer','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Our bio-thermometer is an E.coli that changes color at different temperatures. Such a biological thermometer can for example be applied as a temperature reporter system in large-scale fermentations, or as a temperature-inducible protein production system. A class of known biological temperature sensitive switches are called RNA thermometers. Below a certain temperature, the secondary structure of an RNA thermometer masks the Shine-Dalgarno (SD) sequence in a hairpin. Above this temperature the hairpin opens up, allowing the ribosome to bind . The input temperature is then converted to a color output by switching on the isoprenoid bio-synthetic pathway. ','http://2008.igem.org/Team:TUDelft','TU Delft','2008','They have confirm the functionality of one RNA thermometer (K115035) experimentally. Furthermore, mathematical models describing the biological thermometer were set up. Also software with a graphical user interface was made to design RNA secondary structu',NULL,NULL,NULL),('Eco400_16',3,'Artificial Multi-Cellular Self-organized Syst','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','','http://2008.igem.org/Team:USTC','USTC','2008','',NULL,NULL,NULL),('Eco500_10',3,'Flow-E','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','The final goal of this bacteria is to introduce retinal biosynthesis. The team firstly modified the strain in order to overexpress flagella(increase force generation potential and move furthur.). Then they induced a light sensor into the bacteria chemotaxis pathway which can fastly response. Finally they applied the bacteria to the retinal biosynthesis.','http://2010.igem.org/Team:SDU-Denmark/','SDU-Denmark','2010','First two steps have proved to be partly feasible. The final goal hasn\'t been achieved yet.',NULL,NULL,NULL),('Eco500_11',3,'Synthetic-biological approaches to osteoarthr','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','The project has designed two kinds of cells which can both detect the OA(Osteoarthritis) in human\'s tissue.One is an eukaryotic approach and another is  prokaryotic.Each contains the detector(detect the position of the OA tissue),supervisor(control the spread of the inflammation)as well as the actuator(didifferentiate the lesion\'s tissue.).','http://2010.igem.org/Team:SJTU-BioX-Shanghai','SJTU-BioX-Shanghai','2010','Partly accoumplished their goal.Most achievements belong to the eukaryotic approach.',NULL,NULL,NULL),('Eco500_12',3,'iColi','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Making a multi-pathogen detecting biosensor. A quorum sensing molecule activating a pathway which in turn activates a promoter which will be linked to a reporter molecule initially GFP but in final iColi eChromi will be used to give a variety of visible results','http://2010.igem.org/Team:Sheffield/Project','SHEFFIELD','2010','Can not get the result of their experiments. ',NULL,NULL,NULL),('Eco500_13',3,'Spot on Treatment','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project utilizes a safe form of E.coli which can tightly combine to the lesion\'s skin of Vitiligo patient and cure the disease---maybe in a temporary way.','http://2010.igem.org/Team:Stockholm/','Stockholm  ','2010','This team didn¡¯t show their results obviously. From their results pages, we conjecture that they haven\'t achieve their goals.However,there are still some useful parts you can use.',NULL,NULL,NULL),('Eco500_14',3,'?Lignin Terminator','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This team makes some Saccharomyces cerevisiae which could synthesize lignin degrading enzymes and secrete them out.\nThis design can be applied to the papermaking industry as well as the alcohol fuels production.','http://2010.igem.org/Team:Tianjin/','Tianjin','2010','They have successfully developed an expression system and applied on target enzymes.',NULL,NULL,NULL),('Eco500_15',3,'E.coli fiber project','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Bacteria cellulose(BC) has 1/100 ¡«1/1000 size than plant cellulose. This project tried to produce BC which has a big applicable potential in the future.','http://2010.igem.org/Team:Tokyo_Metropolitan/','Tokyo Metropolitan','2010','They\'ve mentioned that they successfully assembly 9 parts but didn\'t show the experimental result of their projects.',NULL,NULL,NULL),('Eco500_16',3,'Eco Tanker','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project designed an E.coli which can load some substances in the bacteria with a special construct.After inducing the E.colis gathering together in a specific position, they will automatically lysed by aggregation. This project can be applied to the molecule transport.','http://2010.igem.org/Team:Tokyo-NoKoGen','Tokyo NoKoGen','2010','According to their experiment notes, they have successfully achieved all the plannings.',NULL,NULL,NULL),('Eco500_17',3,'E.trojan','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','So as to lower the antibiotic resistance of the bacteria, this team has inserted the T4MO synthesis gene which can produce T4MO---an enzyme which can degrade the indole and cause a quorum-sensing disruption. This effect is helpful to the antibiotic therapies.','http://2011.igem.org/Team:HKUST-Hong_Kong','HKUST-HongKong','2011','successfully proved that the effect of T4MO(degrade indole) and the indole(help to generate the antibiotic resistance).Also achieved the goal of their project.',NULL,NULL,NULL),('Eco500_18',3,'ChloriColight','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','so as to lower the antibiotic resistance of the bacteria, this team has inserted the T4MO synthesis gene which can produce T4MO---an enzyme which can degrade the indole and cause a quorum-sensing disruption. This effect is helpful to the antibiotic therap','http://2011.igem.org/Team:Hong_Kong-CUHK','Hong_Kong-CUHK','2011','They can regulate any gene expression by controlling intracellular Cl- concentration(at that time).The two applications haven\'t been completely accoumplished yet. ',NULL,NULL,NULL),('Eco500_19',3,'Sunscreen','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Under the sunshine, the bacteria can grow faster(due to the energy advantage of Proteorhodopsin(PR) expression) and secrete the  sunscreen ','http://2011.igem.org/Team:IIT_Madras/','IIT_Madras','2011','This is not the main project of the team,but they successfully express the screening as they wishes.They also claimed that this bio-product was very safe.',NULL,NULL,NULL),('Eco500_20',3,'Artemis','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project helps the engineering bacteria to express under low level of carbohydrate stress.The key point is the Carbon stress induced promoter PcstA, which is triggered  in carbonhydrate starvation.As a result, more protein produced for lesser input of carbohydrates, and thereby more economical','http://2011.igem.org/Team:IIT_Madras/Project','IIT_Madras','2011','They finished the main parts of the project and successfully accomplished what they want.They also improve the activity of the promoter PcstA.',NULL,NULL,NULL),('Eco500_21',3,'Tailoring your Avatar','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project focused on a wireless control of remote optogenetic neuro-stimulation with minimizing the invasion and damage to the neurons. They choosed Magnetospirillum magneticum Strain AMB-1 as their foundation.Based on the BRET(bioluminescence resonance energy transfer) Phenomenon and the CHAMP(the computed helical anti-membrane protein) protein.','http://2011.igem.org/Team:NYMU-Taipei/','NYMU-Taipei','2011','Most of the construct have successfully inserted into the AMB-1.  Optomagnetic design shines light on three-dimentional wireless optogenetic stimulation. The experiment is successful.',NULL,NULL,NULL),('Eco500_22',3,'Ecolion','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project developed the Ecolion, which collects heavy metals from the environments,aimed at cleaning up heavy-metal pollution as well as for the mining of valuable metals for industrial applications.  The EcoLion that has stored heavy metals will be conveniently collected by light using phototaxis or self-aggregation.','http://2011.igem.org/Team:Tokyo-NoKoGen','Tokyo-NoToGen','2011','Four devices (Metallothionein, Phototaxis,Aggregation and Lysis) successfully functioned as expected.',NULL,NULL,NULL),('Eco500_23',3,'RPS game','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','Using different signaling molecules corresponding either to rock, paper or scissors.The competitors are human(holding 3 molecules) and E.coli who can randomly express 3 different molecules.There are also a judge E.coli team which has an AND-gate promoter which produces a single output based on two inputs.','http://2011.igem.org/Team:Tokyo_Tech','Tokyo-Tech','2011','1',NULL,NULL,NULL),('Eco900_00',1,'A Synthetic Genetic EdgeDetection Program','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','This project aimed at constructing a genetically encoded edge detection algorithm that programs an isogenic community of E.coli which can sense an image of light. The E.coli can identify the edge of the dark-light edges through a series of communication and as a result present the outcome of the computation visually. An engineered light sensor helps the bacteria to distinguish whether it’s in the dark or light surroundings. Several genetic logic gates in the cell can control the expression conditions that only cells that receive light stimulation as well as the diffusible signal can give a positive output.','Cell (2009)137, 1272–1281',',Howard M. Salis,etc.','2009','','/home/wwwroot/igem2013/img/project/Eco900_00_pro.jpg','Conversion of the edge detection algorithm into a molecular genetic system. (Left) The light-sensitive protein Cph8 is a chimeric sensor kinase bearing the photoreceptor domain of the Synechocystis phytochrome Cph1 and the kinase domain of E.coli EnvZ (Levskaya et al., 2005). Cph8 requires the covalently associated chromophore phycocyanobilin (PCB, blue pentagons) which is produced from heme by the products of the two constitutively expressed genes ho1 and pcyA (Gambetta and Lagarias, 2001). In the presence of red light, the kinase activity of Cph8 is inhibited, precluding the transfer of a phosphoryl group (light green circle) to the response regulator OmpR (orange dumbbell) and subsequent transcription from the ompC promoter (PompC). The dark sensor therefore functions as a NOT light transcriptional logic gate. (Center) luxI and cI are expressed polycistronically from the NOT light gate. LuxI is a biosynthetic enzyme from V.fischeri that produces the cell-cell communication signal 3-oxohexanoyl-homoserine lactone (AHL). CI is the transcriptional repressor protein from phage l. AHL binds to the constitutively expressed transcription factor LuxR to activate expression from the Plux-l promoter while CI dominantly represses it. Plux-l therefore functions as an X AND (NOT Y) transcriptional logic gate. (Right) The output of Plux-l is lacZ, the product of which (b-galactosidase) cleaves a substrate in the media to produce black pigment (Experimental Procedures). The edge detection algorithm is encoded as 10,020 basepairs of DNA, carried on three plasmid backbones (Experimental Procedures). ','/home/wwwroot/igem2013/img/circuit/cct900.jpg'),('Eco900_01',1,'Two- and three-input TALE-based AND logic computation in embryonic stem cells','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','TALEs (Transcriptional activator-like effectors) can be redesigned to target a certain DNA sequence. TALEs can perform Boolean logic computation in mammalian cells. A functional TALE can be reconstituted from two inactive parts by using a split-intein protein-splicing strategy and as a result, generating two-input AND logic computation. The scientists have also demonstrated three-piece intein splicing in mammalian cells and use it to perform three-input AND computation. Comparing construct variants in the same genomic context, they modulated the strength of the TALE-responsive promoter to improve the output of these circuits. Their work establishes split TALEs as a tool for building logic computation with the potential of controlling expression of endogenous genes or transgenes in response to a combination of cellular signals.','Nucl. Acids Res.  August 27, 2013','Florian Lienert, Joseph P. Torella, Jan-Hung Chen, Michael Norsworthy, Ryan R. Richardson and Pamela A. Silver.','2013',NULL,'/home/wwwroot/igem2013/img/project/Eco900_01_pro.png','Engineering of a split TALE-based AND circuit. (A) Schematic of the split TALE reconstitution process. TAL-N:Int-N and Int-C:TAL-C fragments are expressed from CMV expression plasmids. The two split TAL–intein fragments dimerize and undergo protein splicing to reconstitute the full TALE, which binds and activates a CFP reporter containing a binding site for TAL118 upstream of a HSV minimal promoter (m-pr). TAL118 induces expression through a transcriptional activator domain VP64 (24). The N- and C-TAL:Int fragments contain nuclear localization signals and are tagged with co-translationally cleaved t2A:zsYellow and t2A:mCherry, respectively.','/home/wwwroot/igem2013/img/cct/cct901.png'),('Eco900_02',1,'Engineering Escherichia coli to see light','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','In this project, they have designed a bacterial system that is switched between different states by red light. The system consists of a synthetic sensor kinase that allows a lawn of bacteria to function as a biological film, such that the projection of a pattern of light on to the bacteria produces a high-definition (about 100 megapixels per square inch), two-dimensional chemical image. This spatial control of bacterial gene expression could be used to ‘print’ complex biological materials, for example, and to investigate signaling pathways through precise spatial and temporal control of their phosphorylation steps.','Nature 2005, 438:441-442','Levskaya, Aaron A. Chevalier, Jeffrey J. Tabor, Zachary Booth Simpson, Laura A. Lavery, Matthew Levy, Eric A. Davidson, Alexander Scouras, Andrew D. Ellington.','2005',NULL,'/igem2013/img/project/Eco900_02_pro.jpg','The part of the photoreceptor that responds to light, phycocyanobilin, is not naturally produced in E. coli. We therefore introduced two phycocyanobilin-biosynthesis genes(ho1 and pcyA) from Synechocystis that convert haem into phycocyanobilin8 (parts BBa_I15008, BBa_I15009;) Individual Cph1–EnvZ chimaeras were then activated at 37 C for 4 h with broad-spectrum light and assayed for expression of the lacZ reporter. The chimaera Cph8 (BBa_I15010) produced a particularly strong response to light.For bacterial photography, they grew a lawn of bacteria on agar. The LacZ reporter was visualized by addition of S-gal (3,4-cyclohexenoesculetin-D-galactopyranoside): LacZ catalyses the formation of a stable, insoluble, black precipitate from S-gal. Light repressed gene expression in the bacteria, giving a high-contrast replica of the applied image on the biological film, in which light regions appeared light and dark regions were dark (Fig. 1c, and see supplementary information). The LacZ activity showed a graded response to increasing light intensity that was minimal in the brightest light     Parts information:  BBa_I15008, BBa_I15009: Producing photoreceptor phycocyanobilin that responds to light.BBa_I15010: producing a particularly strong response to light.','/home/wwwroot/igem2013/img/cct/cct902.jpg'),('Eco900_03',1,'Biological Signal Processing with a Genetic Toggle Switch','SUSTC-ShenzhenB','igem2013@sustc.edu.cn','(A) Phase diagram of the toggle switch as a function of the effective maximal transcription rates ~nA and ~nB. Here, the parameters are chosen such that the system is in the bistable regime in the absence of input signals (point O) and the circuit is set to the ON state initially. The curves indicate dynamic changes of ~nA and ~nB, incurred by applying the toggle signal (simultaneous expression of both input genes J and K) for 15 min (green solid curve) and then releasing it (red curve) or by applying it continuously (green dotted curve). (B) The same trajectories in the (½A2tot,½B2tot)-plane. In (A) and (B) the system quickly approaches the stable fixed point T, representing a low A /low B state, such that a release of the toggle signal will lead to an unpredictable outcome dominated by molecular noise.','journal.pone.0068345','Patrick Hillenbrand, Georg Fritz, Ulrich Gerland1*','2013',NULL,'/igem2013/img/project/Eco900_03_pro.jpg','Possible realization of a genetic JK-latch, in which the AND gates are implemented by heterodimerization between the input proteins J and K and the proteins of the memory element A and B. The memory element (SR-latch) is translated into a genetic toggle switch, which can be set to ON (A high, B low) and reset to OFF (A low, B high) by additional repressor sites, binding the heterodimers JB and KA, respectively. Parts information: Each promoter can be repressed by certain chemicals','/home/wwwroot/igem2013/img/cct/cct903.jpg');
/*!40000 ALTER TABLE `basicinfo` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `biobrickinfo`
--

DROP TABLE IF EXISTS `biobrickinfo`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `biobrickinfo` (
  `codingframeid` int(4) NOT NULL,
  `order` int(1) NOT NULL,
  `biobrickID` varchar(5) NOT NULL,
  `biobrick` varchar(30) NOT NULL,
  `DNAproperty` varchar(25) NOT NULL,
  `function` varchar(100) DEFAULT NULL,
  PRIMARY KEY (`biobrickID`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `biobrickinfo`
--

LOCK TABLES `biobrickinfo` WRITE;
/*!40000 ALTER TABLE `biobrickinfo` DISABLE KEYS */;
INSERT INTO `biobrickinfo` VALUES (3,1,'00011','promoter Lacl(R00010)','promoter',''),(3,2,'00012','RBS(B0034)','RBS coding',''),(3,3,'00013','alsS(K539713)','protein coding','Alpha-acetolactate synthase protein'),(3,4,'00014','RBS(B0034)','RBS coding',''),(3,5,'00015','livC(K539711)','protein coding','Ketolacid reductoisomerase,NAD(P)-binding'),(3,6,'00016','RBS(B0034)','RBS coding',''),(3,7,'00017','livD(K539712)','protein coding','Dihydroxyacid dehydratase'),(3,8,'00018','37¡ãC RBS(K115002)','RBS coding',''),(3,9,'00019','tetR(C0040)','protein coding','TetR'),(3,10,'00020','terminator(J61048)','terminator coding',''),(3,11,'00021','promoter PtetR(R0040)','promoter',''),(3,12,'00022','RBS(B0032)','RBS coding',''),(3,13,'00023','kivd(K539714)','protein coding','Alpha-ketoisovalerate decarboxylase'),(3,14,'00024','terminator(B0010)','terminator coding',''),(3,15,'00025','terminator(B0012)','terminator coding',''),(4,1,'00026','promoter Pconst','promoter',''),(4,2,'00027','RBS(B0034)','RBS coding',''),(4,3,'00028','fadR(K817001)','protein coding','FadR'),(4,4,'00029','terminator','terminator coding',''),(5,1,'00030','promoter PfadBA(K817002)','promoter',''),(5,2,'00031','RBS(B0031)','RBS coding',''),(5,3,'00032','tetR(C0040)','protein coding','TetR'),(5,4,'00033','RBS','RBS coding',''),(5,5,'00034','LuxI(C0061)','protein coding','LuxI'),(5,6,'00035','terminator(B0010)','terminator coding',''),(5,7,'00036','terminator(B0012)','terminator coding',''),(5,8,'00037','promoter PtetR(R0040)','promoter',''),(5,9,'00038','RBS(B0034)','RBS coding',''),(5,10,'00039','lacI(C0012)','protein coding','LacI'),(5,11,'00040','terminator(B0010)','terminator coding',''),(5,12,'00041','terminator(B0012)','terminator coding',''),(5,13,'00042','promoter PLacI(R0010)','promoter',''),(5,14,'00043','RBS','RBS coding',''),(5,15,'00044','glp-1','protein coding','Glp-1'),(5,16,'00045','RBS','RBS coding',''),(5,17,'00046','cpp','protein coding','CPP'),(5,18,'00047','terminator','RBS coding',''),(6,1,'00048','promoter Pconst','promoter',''),(6,2,'00049','RBS','RBS coding',''),(6,3,'00050','LuxR(C0062)','protein coding','LuxR'),(6,4,'00051','terminator','terminator coding',''),(6,5,'00052','promoter pLuxR','promoter',''),(6,6,'00053','RBS','RBS coding',''),(6,7,'00054','tetR(C0040)','protein coding','TetR'),(6,8,'00055','terminator','terminator coding',''),(7,1,'00056','promoter Pconst','promoter',''),(7,2,'00057','RBS','RBS coding',''),(7,3,'00058','Clts(K098988)','protein coding','Clts(temperature-sensitive cI repressor)'),(7,4,'00059','promoter Pcl','promoter',''),(7,5,'00060','RBS','RBS coding',''),(7,6,'00061','glp-1','protein coding','Glp-1'),(7,7,'00062','RBS','RBS coding',''),(7,8,'00063','cpp','protein coding','CPP'),(7,9,'00064','terminator','terminator coding',''),(8,1,'00065','promoter Pbad(K821004)','promoter',''),(8,2,'00066','RBS','RBS coding',''),(8,3,'00067','Light sensor(K785004)','protein coding','Light sensor'),(8,4,'00068','terminator1','terminator coding',''),(9,1,'00069','promoter PtetR(R0040)','promoter',''),(9,2,'00070','tetO','protein coding','tetO'),(9,3,'00071','RBS','RBS coding',''),(9,4,'00072','Lux(K785003)','protein coding','GFP(green fluorescent protein)'),(9,5,'00073','terminator2','terminator coding',''),(10,1,'00074','promoter PycgZ','promoter',''),(10,2,'00075','RBS','RBS coding',''),(10,3,'00076','tetR(C0040)','protein coding','TetR'),(10,4,'00077','terminator1','terminator coding',''),(11,1,'00078','promoter PycgZ','promoter',''),(11,2,'00079','RBS','RBS coding',''),(11,3,'00080','Lux(K785003)','protein coding','GFP(green fluorescent protein)'),(11,4,'00081','terminator2','terminator coding',''),(12,1,'00082','promoter PtetR(R0040)','promoter',''),(12,2,'00083','RBS(B0034)','RBS coding',''),(12,3,'00084','phaC(K342001)','protein coding','P(3HB) synthase'),(12,4,'00085','RBS(B0034)','RBS coding',''),(12,5,'00086','phaA(K759004)','protein coding','phaA(acetyl-CoA acetyltransferase)'),(12,6,'00087','RBS(B0034)','RBS coding',''),(12,7,'00088','phaB(K759005)','protein coding','phaB(acetyl-CoA reductase)'),(12,8,'00089','terminator(B0015)','terminator coding',''),(13,1,'00090','pBAD+Arac(I0050)','promoter',''),(13,2,'00091','RBS(B0034)','RBS coding',''),(13,3,'00092','Ag43(K346007)','protein coding','Ag43'),(13,4,'00093','terminatro(B0015)','terminator coding',''),(14,1,'00094','pBAD+Arac(I0050)','promoter',''),(14,2,'00095','RBS(B0034)','RBS coding',''),(14,3,'00096','eCFP(E0020)','protein coding','eCFP(Cyan Fluorescent protein)'),(14,4,'00097','RBS(B0034)','RBS coding',''),(14,5,'00098','Ag43(K346007)','protein coding','Ag43'),(14,6,'00099','terminatro(B0015)','terminator coding',''),(15,1,'00100','Ptrc(R0011)','promoter',''),(15,2,'00101','RBS(B0034)','RBS coding',''),(15,3,'00102','yddV(K748003)','protein coding','yddV '),(15,4,'00103','terminator(B0010)','terminator coding',''),(15,5,'00104','terminator(B0012)','terminator coding',''),(16,1,'00105','promoter T5','promoter',''),(16,2,'00106','RBS','RBS coding',''),(16,3,'00107','LuxR(C0063)','protein coding','LuxR'),(16,4,'00108','terminator','terminator coding',''),(17,1,'00109','promoter pLuxR','promoter',''),(17,2,'00110','RBS','RBS coding',''),(17,3,'00111','lys(K748002)','protein coding','lysostaphin protein'),(17,4,'00112','marR','protein coding','marR'),(17,5,'00113','terminator','terminator coding',''),(18,1,'00114','promoter pmerT','promoter',''),(18,2,'00115','RBS','RBS coding',''),(18,3,'00116','lysis E7','protein coding','lysis E7'),(18,4,'00117','terminator','terminator coding',''),(19,1,'00118','promoter T5','promoter',''),(19,2,'00119','RBS','RBS coding',''),(19,3,'00120','agrC(K748001)','protein coding','agrC'),(19,4,'00121','agrA(K748000)','protein coding','agrA'),(19,5,'00122','terminator','terminator coding',''),(20,1,'00123','promoter P2(K748008)','promoter',''),(20,2,'00124','RBS(B0034)','RBS coding',''),(20,3,'00125','GFP(E0040)','protein coding','GFP(green fluorescent protein)'),(20,4,'00126','LuxI(C0061)','protein coding','LuxI'),(20,5,'00127','terminator(B0010)','terminator coding',''),(20,6,'00128','terminator(B0012)','terminator coding',''),(21,1,'00129','Ptrc(R0011)','promoter',''),(21,2,'00130','RBS(B0034)','RBS coding',''),(21,3,'00131','yddV(K748003)','protein coding','yddV '),(21,4,'00132','terminator(B0010)','terminator coding',''),(21,5,'00133','terminator(B0012)','terminator coding',''),(1009,1,'10077','Laclq(Q04121)','promoter','IPTG activates Laclq and promotes transcription'),(1009,2,'10078','RBS(B0034)','RBS coding',''),(1009,3,'10079','YF1(K592004)','protein coding','blue-light sensor protein'),(1009,4,'10080','FixJ(K592005)','protein coding','the response regulator to YF1.FixJ in turn can be regulated by YF1.'),(1009,5,'10081','pFixK2(K592006)','promoter','This promoter shows very little leaky activity in the absence of FixJ.'),(1009,6,'10082','Cl(Q04510)','protein coding','express cl repressor protein'),(1009,7,'10083','pR promoter','promoter','repressed by cl protein'),(1009,8,'10084','ClyA','protein coding','Cytolysin A pore forming protein causes cell lysis.'),(1009,9,'10085','Terminator(B0014)','terminator coding',''),(1010,1,'10086','T7 promoter','promoter','constitutive promoter derived from the T7 bacteriophage.Perhaps its highest output temperature is 25'),(1010,2,'10087','StrongRBS(J15001)','RBS coding',''),(1010,3,'10088','INPNC','protein coding','Ice Nucleation Protein(N'),(1010,4,'10089','StrongRBS(J15001)','RBS coding',''),(1010,5,'10090','DARPin(K404314)','protein coding','DARPin ,a kind of an antibody-mimetic protein, has a high affinity to HER2.'),(1010,6,'10091','Terminator(B0014)','terminator coding',''),(2000,1,'20000','P(Tetr)(K247003)','promoter','This promoter is constitutively ON and repressed by TetR'),(2000,2,'20001','RBS(K247003)','RBS coding',''),(2000,3,'20002','LuxR(K247003)','protein coding','Express LuxR protein to form complex to activate the next promoter'),(2000,4,'20003','Terminator(B0014)','terminator coding',''),(2001,1,'20004','Promoter LuxR(K247004)','promoter','The AHL LuxR complex dimer activate this promoter to express gene of downstream'),(2001,2,'20005','RBS(B0034)','RBS coding',''),(2001,3,'20006','GFP(E0040)','protein coding','Express GFP to show the time'),(2001,4,'20007','Terminator(B0014)','terminator coding',''),(2002,1,'20008','P(Tetr)(R0040)','promoter','This promoter is constitutively ON and repressed by TetR'),(2002,2,'20009','RBS(B0034)','RBS coding',''),(2002,3,'20010','LuxI(C0161)','protein coding','Produce 3OC6HSL as a stimulus'),(2002,4,'20011','Terminator(B0010)','terminator coding',''),(2002,5,'20012','Terminator(B0012)','terminator coding',''),(2003,1,'20013','P(Tetr)(R0040)','promoter','This promoter is constitutively ON and repressed by TetR'),(2003,2,'20014','RBS(B0034)','RBS coding',''),(2003,3,'20015','LuxR(C0062)','protein coding','Receive signal in the form of Lux-signaling system AHL and express LuxR protein to activate the down'),(2003,4,'20016','Terminator(B0010)','terminator coding',''),(2003,5,'20017','Terminator(B0012)','terminator coding',''),(2003,6,'20018','P(Lux)(R0062)','promoter','This promoter is activated by LuxR protein produced in the upstream'),(2003,7,'20019','RBS(B0034)','RBS coding',''),(2003,8,'20020','GFP(E0040)','protein coding','Express GFP to \"paint\"'),(2003,9,'20021','Terminator(B0010)','terminator coding',''),(2003,10,'20022','Terminator(B0012)','terminator coding',''),(2010,1,'20055','?delta sigma 70 promoter','promoter',''),(2010,2,'20056','LuxP(K218000)','protein coding','Express LuxP protein to form AI-2 LuxP complex'),(2010,3,'20057','LuxQ(K218000)','protein coding','Express histidine kinase to stimulate the formation of AI-2 LuxP'),(2010,4,'20058','p(tetR)(R0040)','promoter',''),(2010,5,'20059','LuxO(K218001)','protein coding','DNA binding response regulator protein LuxO'),(2010,6,'20060','LuxU(K218001)','protein coding','Express cytoplasmic protein LuxU to be phosphated'),(2010,7,'20061','Terminator(B0015)','terminator coding',''),(2011,1,'20062','qrr4 promoter(K131017)','promoter',''),(2011,2,'20063','RBS(B0034)','RBS coding',''),(2011,3,'20064','GFP(E0040)','protein coding','Express GFP'),(2011,4,'20065','Terminator(B0010)','terminator coding',''),(2011,5,'20066','Terminator(B0012)','terminator coding',''),(2012,1,'20067','qrr4 promoter(K131017)','promoter',''),(2012,2,'20068','RBS(B0034)','RBS coding',''),(2012,3,'20069','c1 lamda(Q04510)','protein coding','Express cI lamda repressor protein'),(2012,4,'20070','Terminator(B0010)','terminator coding',''),(2012,5,'20071','Terminator(B0012)','terminator coding',''),(2012,6,'20072','pC1 lamda','promoter',''),(2012,7,'20073','RBS(B0034)','RBS coding',''),(2012,8,'20074','aiiA(C0160)','protein coding','AHL-degrading enzyme'),(2012,9,'20075','Terminator(B0010)','terminator coding',''),(2012,10,'20076','Terminator(B0012)','terminator coding',''),(2015,1,'20089','LacI promoter(R0010)','promoter',''),(2015,2,'20090','RBS(B0030)','RBS coding',''),(2015,3,'20091','LovTAP(K191006)','protein coding','Express LovTAP hybrid protein that is sensitive to light to be a signal'),(2015,4,'20092','Terminator(B0010)','terminator coding',''),(2015,5,'20093','Terminator(B0012)','terminator coding',''),(2016,1,'20094','TrpO(K191007)','promoter',''),(2016,2,'20095','RBS(B0034)','RBS coding',''),(2016,3,'20096','Target gene','protein coding','Express Gene of interest'),(2016,4,'20097','Terminator(B0010)','terminator coding',''),(2016,5,'20098','Terminator(B0012)','terminator coding',''),(2017,1,'20099','TrpO(K191007)','promoter',''),(2017,2,'20100','RBS(B0034)','RBS coding',''),(2017,3,'20101','TetR(C0040)','protein coding','Express TetR protein to repress the downstream promoter'),(2017,4,'20102','Terminator(B0010)','terminator coding',''),(2017,5,'20103','Terminator(B0012)','terminator coding',''),(2017,6,'20104','p(tetR)(R0040)','promoter','Repressed by TetR protein'),(2017,7,'20105','RBS(B0034)','RBS coding',''),(2017,8,'20106','Target gene','protein coding','Express Gene of interest'),(2017,9,'20107','Terminator(B0010)','terminator coding',''),(2017,10,'20108','Terminator(B0012)','terminator coding',''),(2018,1,'20109','P_araC/lacI','promoter','The ParaC/lacI promoter is enhanced by araC and repressed by lacI.'),(2018,2,'20110','AraC','protein coding','Produce AraC protein to activate promoters'),(2018,3,'20111','P_araC/lacI','promoter','The ParaC/lacI promoter is enhanced by araC and repressed by lacI.'),(2018,4,'20112','cIII','protein coding','Produce cIII protein to repress Hfl'),(2019,1,'20113','P_araC/lacI','promoter','The ParaC/lacI promoter is enhanced by araC and repressed by lacI.'),(2019,2,'20114','LacI','protein coding','Produce LacI protein to repress promoters'),(2020,1,'20115','LacI','protein coding','Express the LacI protein to repress the oscillator'),(2020,2,'20116','cI','protein coding','Produce cI protein to repress the cro'),(2020,3,'20117','Operator','protein coding',''),(2020,4,'20118','cro','protein coding','Produce cro protein to repress the cI'),(2020,5,'20119','cII','protein coding','Produce cII protein to stimulate the cI'),(4000,1,'40000','Pc(J2310)','promoter',''),(4000,2,'40001','RBS(B0034)','RBS coding',''),(4000,3,'40002','luxR(C0062)','protein coding','coding luxR(C0062) protein'),(4000,4,'40003','Terminator','terminator coding',''),(4001,1,'40004','Plux(R0062)','promoter','repressed by CI434 protein'),(4001,2,'40005','RBS(B0034)','RBS coding',''),(4001,3,'40006','CI434(C0052)','protein coding','coding CI434(C0052) protein'),(4001,4,'40007','Terminator','terminator coding',''),(4002,1,'40008','Plux/CI434(K396002)','promoter','activate by luxR&repressed by CI434  '),(4002,2,'40009','RBS(B0034)','RBS coding',''),(4002,3,'40010','T7 RNAP(K145001)','protein coding','coding T7 RNAP(K145001)'),(4002,4,'40011','Terminator','terminator coding',''),(4003,1,'40012','Plux(R0062)','promoter','repressed by CI434 protein'),(4003,2,'40013','RBS(B0034)','RBS coding',''),(4003,3,'40014','tetR(C0040)','protein coding','coding tetR(C0040) protein'),(4003,4,'40015','Terminator','terminator coding',''),(4004,1,'40016','Ptet(R0040)','promoter','tetR replessed'),(4004,2,'40017','RBS(B0034)','RBS coding',''),(4004,3,'40018','cI(C0051)','protein coding','coding CI(C0051) protein'),(4004,4,'40019','Terminator','terminator coding',''),(4005,1,'40020','PT7/cI(K39600)','promoter','activate by T7 RNAP&repressed by CI '),(4005,2,'40021','RBS(B0034)','RBS coding',''),(4005,3,'40022','GFP(E0040)','protein coding','coding GFP protein'),(4005,4,'40023','Terminator','terminator coding',''),(4006,1,'40024','Pc(J23116)','promoter',''),(4006,2,'40025','RBS(B0034)','RBS coding',''),(4006,3,'40026','luxR(C0062)','protein coding','coding luxR(C0062) protein'),(4006,4,'40027','Terminator(B0010)','terminator coding',''),(4006,5,'40028','Terminator(B0012)','terminator coding',''),(4007,1,'40029','Plux(R0062)','promoter','repressed by CI434 protein'),(4007,2,'40030','RBS(B0034)','RBS coding',''),(4007,3,'40031','T7ptag(K228000)','protein coding','coding T7 RNA polymeraze'),(4007,4,'40032','LacI(C0010)   ','protein coding','coding LacI(C0010) protein'),(4007,5,'40033','cI(C0051)','protein coding','coding CI(C0051) protein'),(4007,6,'40034','Terminator','terminator coding',''),(4008,1,'40035','Plac(R0010)','promoter','activate by LacI'),(4008,2,'40036','RBS(B0034)','RBS coding',''),(4008,3,'40037','SupD(K228001)','protein coding','coding SupD protein'),(4008,4,'40038','Terminator(B0012)','terminator coding',''),(4009,1,'40039','PT7/cI(K39600)','promoter',''),(4009,2,'40040','RBS(B0034)','RBS coding',''),(4009,3,'40041','GFP(E0040)','protein coding','coding GFP protein'),(4009,4,'40042','Terminator(B0015)','terminator coding',''),(4010,1,'40043','promoter','promoter',''),(4010,2,'40044','RBS','RBS coding',''),(4010,3,'40045','Lignin Peroxidase(144000)','protein coding','coding lignin peroxidase '),(4010,4,'40046','Terminator','terminator coding',''),(4011,1,'40047','PtetR(R0040)','promoter','tetR replessed'),(4011,2,'40048','RBS(B0034)','RBS coding',''),(4011,3,'40049','GFP(E0040)','protein coding','coding GFP protein'),(4011,4,'40050','Terminator(B0010)','terminator coding',''),(4011,5,'40051','Terminator(B0012)','terminator coding',''),(9000,1,'90000','PompC ','promoter','can be equal to red light repressed promoter'),(9000,2,'90001','B0034','RBS coding','strong ribosome binding sites'),(9000,3,'90002','luxI','protein coding','AHL'),(9000,4,'90003','cI','protein coding','transcriptional repressor'),(9000,5,'90004','B0015','terminator coding','rrnB/TE tandem transcription terminator'),(9000,6,'90005','R0040','promoter','PLtet-O1 constitutive promoter in the absense of Tet repressor'),(9000,7,'90006','luxR','protein coding','transcription factor luxR'),(9000,8,'90007','B0015','terminator coding','rrnB/TE tandem transcription terminator'),(9000,9,'90008','Plux-lamda','promoter','LuxR::AHL activated, cI repressed promoter'),(9000,10,'90009','LacZ','proterin coding','b-galactosidase'),(9000,11,'90010','B0015','terminator coding','rrnB/TE tandem transcription terminator'),(9001,1,'90011','R0040','promoter','LuxR::AHL activated, cI repressed promoter'),(9001,2,'90012','cph8','protein coding','A light-sensitive protein'),(9001,3,'90013','B0015','terminator coding','rrnB/TE tandem transcription terminator'),(9002,1,'90014','R0040','promoter','PLtet-O1 constitutive promoter in the absense of Tet repressor'),(9002,2,'90015','ho1','protein coding','precursor of chromophore phycocyanobilin'),(9002,3,'90016','pcyA','protein coding','precursor of chromophore phycocyanobilin'),(9002,4,'90017','B0015','terminator coding ','rrnB/TE tandem transcription terminator');
/*!40000 ALTER TABLE `biobrickinfo` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `cctframe`
--

DROP TABLE IF EXISTS `cctframe`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `cctframe` (
  `circuitno` varchar(7) NOT NULL,
  `codingframeid` int(4) NOT NULL,
  PRIMARY KEY (`codingframeid`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `cctframe`
--

LOCK TABLES `cctframe` WRITE;
/*!40000 ALTER TABLE `cctframe` DISABLE KEYS */;
INSERT INTO `cctframe` VALUES ('cct_002',3),('cct_003',4),('cct_003',5),('cct_003',6),('cct_004',7),('cct_005',8),('cct_005',9),('cct_005',10),('cct_005',11),('cct_006',12),('cct_007',13),('cct_007',14),('cct_008',15),('cct_008',16),('cct_008',17),('cct_008',18),('cct_008',19),('cct_008',20),('cct_008',21),('cct_108',1009),('cct_108',1010),('cct_200',2000),('cct_200',2001),('cct_201',2002),('cct_202',2003),('cct_204',2010),('cct_204',2011),('cct_204',2012),('cct_206',2015),('cct_206',2016),('cct_206',2017),('cct_207',2018),('cct_207',2019),('cct_208',2020),('cct_400',4000),('cct_400',4001),('cct_400',4002),('cct_400',4003),('cct_400',4004),('cct_400',4005),('cct_401',4006),('cct_401',4007),('cct_401',4008),('cct_401',4009),('cct_402',4010),('cct_403',4011),('cct_900',9000),('cct_900',9001),('cct_900',9002);
/*!40000 ALTER TABLE `cctframe` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `ecocct`
--

DROP TABLE IF EXISTS `ecocct`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `ecocct` (
  `Ecoid` varchar(9) NOT NULL,
  `circuitno` varchar(7) NOT NULL,
  PRIMARY KEY (`Ecoid`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `ecocct`
--

LOCK TABLES `ecocct` WRITE;
/*!40000 ALTER TABLE `ecocct` DISABLE KEYS */;
INSERT INTO `ecocct` VALUES ('Eco000_09','cct_002'),('Eco000_10','cct_003'),('Eco000_11','cct_004'),('Eco000_12','cct_005'),('Eco000_13','cct_006'),('Eco000_14','cct_007'),('Eco000_23','cct_008'),('Eco000_24','cct_008'),('Eco000_58','cct_108'),('Eco200_00','cct_200'),('Eco200_04','cct_204'),('Eco200_06','cct_206'),('Eco200_11','cct_207'),('Eco400_00','cct_400'),('Eco400_01','cct_401'),('Eco400_05','cct_402'),('Eco400_14','cct_403'),('Eco900_00','cct_900');
/*!40000 ALTER TABLE `ecocct` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `expression`
--

DROP TABLE IF EXISTS `expression`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `expression` (
  `codingframeID` int(4) NOT NULL,
  `order` int(1) NOT NULL,
  `biobrickID` varchar(5) NOT NULL,
  `substanceno` varchar(6) NOT NULL,
  PRIMARY KEY (`biobrickID`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `expression`
--

LOCK TABLES `expression` WRITE;
/*!40000 ALTER TABLE `expression` DISABLE KEYS */;
INSERT INTO `expression` VALUES (3,3,'00013','S00003'),(3,5,'00015','S00004'),(3,7,'00017','S00005'),(3,9,'00019','S00006'),(3,13,'00023','S00007'),(4,3,'00028','S00008'),(5,3,'00032','S00006'),(5,5,'00034','S00009'),(5,10,'00039','S00010'),(5,15,'00044','S00011'),(5,17,'00046','S00012'),(6,3,'00050','S00013'),(6,7,'00054','S00006'),(7,3,'00058','S00018'),(7,6,'00061','S00011'),(7,8,'00063','S00012'),(8,3,'00067','S00021'),(9,2,'00070','S00023'),(9,4,'00072','S00022'),(10,3,'00076','S00006'),(11,3,'00080','S00022'),(12,3,'00084','S00025'),(12,5,'00086','S00026'),(12,7,'00088','S00027'),(13,3,'00092','S00028'),(14,3,'00096','S00029'),(14,5,'00098','S00028'),(15,3,'00102','S00036'),(16,3,'00107','S00013'),(17,3,'00111','S00031'),(17,4,'00112','S00032'),(18,3,'00116','S00033'),(19,3,'00120','S00034'),(19,4,'00121','S00035'),(20,3,'00125','S00022'),(20,4,'00126','S00009'),(21,3,'00131','S00036'),(1009,3,'10079','S10000'),(1009,4,'10080','S10001'),(1009,6,'10082','S10002'),(1009,8,'10084','S10004'),(2000,3,'20002','S20001'),(2002,3,'20010','S20003'),(2003,3,'20015','S20001'),(2010,2,'20056','S20013'),(2010,3,'20057','S20011'),(2010,5,'20059','S20014'),(2010,6,'20060','S20012'),(2018,2,'20110','S20005'),(2018,4,'20112','S20025'),(2019,2,'20114','S20021'),(2020,1,'20115','S20021'),(2020,2,'20116','S20022'),(2020,4,'20118','S20023'),(2020,5,'20119','S20024'),(4000,3,'40002','S40001'),(4001,3,'40006','S40002'),(4002,3,'40010','S40003'),(4003,3,'40014','S40004'),(4004,3,'40018','S40005'),(4006,3,'40026','S40001'),(4007,3,'40031','S40006'),(4007,4,'40032','S40007'),(4008,3,'40037','S40008'),(4005,3,'40041','S40000'),(4010,3,'40045','S40009'),(4011,3,'40049','S40010'),(9000,3,'90002','S90000'),(9000,4,'90003','S90001'),(9000,7,'90006','S90004'),(9000,10,'90009','S90010'),(9001,2,'90012','S90005'),(9002,2,'90015','S90002'),(9002,3,'90016','S90003');
/*!40000 ALTER TABLE `expression` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `parts`
--

DROP TABLE IF EXISTS `parts`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `parts` (
  `Ecoid` varchar(9) NOT NULL,
  `designable` int(1) NOT NULL,
  `parts` varchar(200) NOT NULL,
  PRIMARY KEY (`Ecoid`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `parts`
--

LOCK TABLES `parts` WRITE;
/*!40000 ALTER TABLE `parts` DISABLE KEYS */;
INSERT INTO `parts` VALUES ('Eco000_09',1,''),('Eco000_10',1,''),('Eco000_11',1,''),('Eco000_12',1,''),('Eco000_13',1,''),('Eco000_14',1,''),('Eco000_15',0,'K819005/K819006/K819007'),('Eco000_16',0,'K897720/K897624/K897318/S05053/S05054/S05055'),('Eco000_17',0,'K766001/K766002/K766003/K766000/K766005/ K766006/K766007/K766004'),('Eco000_18',0,'K934001/K934012/K934022'),('Eco000_19',0,'K741001/K741002/K741003/K741004/K741008/ K741006/K741009/K741010/K741011/K741012/K741014/K741015/K741016/ K741018/K741019/K741020/K741021'),('Eco000_20',0,'K750000/K750009/K750111'),('Eco000_21',0,'K923001/K923002/K923003'),('Eco000_22',0,'K826007'),('Eco000_23',1,''),('Eco000_24',1,''),('Eco000_25',0,'K737006/K737010/K737045/K737046/K737058'),('Eco000_26',0,'K896000/K896001/K896002'),('Eco000_27',0,'K080010'),('Eco000_28',0,'K124003/K124014/K124017'),('Eco000_29',0,'K137002/K137126/S03970 /S03971/S03973/S04107/S04108/S04109/S04122/S04123/S04110/S04111/S04112/S04041/S04022/S04113/S04054/S04055/K137125/S04114/S04105/S04106/K137131/K137132'),('Eco000_30',0,'K091206/K091146/K091204/S03969'),('Eco000_31',0,'K118023/K118022/K118028/K118015/K118016/K118018/K118017/S03969'),('Eco000_32',0,'K118000/K118002/K118008/K118003/K118001'),('Eco000_33',0,'K118025/K118010/K118024'),('Eco000_34',0,'K152001/K152002/K152003/K152005/K152006/K152009'),('Eco000_35',0,'K098010/K098011/K098984/K098987/K098988/K098989/K098991/K098993/K098994/K098995/K098999'),('Eco000_36',0,'K145001/K145013/K145014/K145015/K145151/K145201/K145205/K145210/K145215/K145217/K145220/K145225/K145230/K145254/K145269/K145270/K145273/K145279/K145280/K145300/K145301/K145302/K145303'),('Eco000_37',0,'K119000/K119001/K119002/K119003/K119004/K119005/K119007/K119008/K119009/K119010/K119011'),('Eco000_58',1,'K811000/K811001/K811002/K811003/K811004'),('Eco200_00',1,'K247003/K247004'),('Eco200_04',1,'K218000/K218001/K131017/Q04510/C0160'),('Eco200_07',0,'K372001/K372003'),('Eco200_08',0,'K316003'),('Eco200_09',0,'K362001/K362005/K362007'),('Eco200_11',1,''),('Eco200_13',0,'K179002'),('Eco200_14',0,'K179004/K179003/S04318'),('Eco200_16',0,'283000/283001K/283002/K283014'),('Eco200_19',0,'K206000/K206001/I13453/K206010/K206009/K206012/K206016'),('Eco200_20',0,'K182005/K182102/K182200/I13600'),('Eco400_00',1,''),('Eco400_01',1,''),('Eco400_02',0,'lacI promoter + Key3C\npTet->locK3d->GFP->Ter'),('Eco400_03',0,''),('Eco400_04',0,''),('Eco400_05',1,'K144000'),('Eco400_06',0,'I764001'),('Eco400_07',0,'K128000/K128001/K128002/K128003/K128004/ K128005/K128006'),('Eco400_08',0,'K120000~K120024'),('Eco400_09',0,'K100000/K100001/K100002/K100003/K100004/K100005/K100006/K100007/K100010'),('Eco400_10',0,'K366000/K366001'),('Eco400_11',0,'K160000/K160001 \n'),('Eco400_12',0,'K154000'),('Eco400_13',0,'K122000/K122002/K122017/K122003/K122004/ K122013/K122018/K122008/K122014/K122001/ K122005/K122010/K122012/K122015/K122021/K122019/K122007/K122006'),('Eco400_14',1,'K121010/K121011/K121012/K121013/K121014/K121015'),('Eco400_15',0,'K115035/K115001/K115002/K115003/K115016/K115017'),('Eco500_10',0,'K343000/K343001/K343003/K343004/K343005/K343006/K343100/K343007/K343002'),('Eco500_11',0,'K387003/K387006/K38700/K387011/K387012'),('Eco500_12',0,'K386003/K386005/K38607'),('Eco500_13',0,'K380000/K380001/K380002/K380003/K380004/K380005/K380006/K380007/K380008K380009/K380010'),('Eco500_14',0,'K500000/K500001/K500002/K500003/K500004K/500005/K500006/K500007/K500008/K500009/K500010/500011/K500012/K500013'),('Eco500_15',0,'K336048/K336060/K336061/K336020/K336021'),('Eco500_16',0,'K317008/K317018/K317023/K317024/K317026/K317028/K317039/K317040'),('Eco500_17',0,'K524000/K524001/K524002/K524003/K524004/K524100/K524005'),('Eco500_18',0,'K559010/K559000/K559011'),('Eco500_19',0,'K572005/K572006/K572007/K572008/K572009/K572010/K572012'),('Eco500_20',0,'K572001/K572005/K572009'),('Eco500_21',0,'K624006/K624010/K624027/K624031/K624032'),('Eco500_22',0,'K519010/K519013/K519001/K519021/K519014/K519020'),('Eco500_23',0,'I751101/K649201/K649202/K649105'),('Eco900_00',1,'J13023/I15022'),('Eco900_01',0,'null'),('Eco900_02',0,'I15010/I15008/I15009'),('Eco900_03',0,'null');
/*!40000 ALTER TABLE `parts` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `protocol`
--

DROP TABLE IF EXISTS `protocol`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `protocol` (
  `Ecoid` varchar(9) NOT NULL,
  `type` varchar(15) NOT NULL,
  `content` text,
  `order` int(11) NOT NULL,
  `id` int(11) NOT NULL AUTO_INCREMENT,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=158 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `protocol`
--

LOCK TABLES `protocol` WRITE;
/*!40000 ALTER TABLE `protocol` DISABLE KEYS */;
INSERT INTO `protocol` VALUES ('Eco900_00','equipment','37ºC shaking incubator',1,1),('Eco900_00','equipment','Benchtop Microcentrifuge ',2,2),('Eco900_00','equipment','Stirplate',3,3),('Eco900_00','equipment','42ºC heat block with 50mL conical adapter ',4,4),('Eco900_00','equipment','36ºC still incubator with hole in top',5,5),('Eco900_00','equipment','Kodak Ektagraphic projector (or other broad wavelength light source)',6,6),('Eco900_00','equipment','34x24mm slide with printed image ',7,7),('Eco900_00','equipment','650nm bandpass filter (Edmund Optics catalogue #43-189)',8,8),('Eco900_00','equipment','Focusing Lens ',9,9),('Eco900_00','material',' LB + 0.1M HEPES pH = 6.6 ',1,10),('Eco900_00','material',' LB + 0.1M HEPES pH = 8.0 ',2,11),('Eco900_00','material',' LB + 0.1M HEPES pH = 6.6 + 0.3mg/mL Sgal + 0.5mg/mL Ferric Ammonium Citrate + 1% Seaplaque agarose ',3,12),('Eco900_00','procedure','1.Prepare 2 solutions of LB media supplemented with 0.1M HEPES buffer. pH one batch ',1,13),('Eco900_00','procedure','2. Inoculate 3mL LB + 0.1M HEPES pH = 8.0 + 50ug/mL Ampicillin, 50ug/mL Kanamycin and 34ug/mL Chloramphenicol with a colony of JW3367c transformed with plasmids pEDL3, pPLPCB and pCph8. ',2,14),('Eco900_00','procedure','3. Grow in 37ºC shaking incubator (250rpm) to saturation (OD600 ~ 3.0-3.7; approximately 14 hours). ',3,15),('Eco900_00','procedure','4. Add 50mg Ferric Ammonium Citrate (Sigma #15-1030), 30mg Sgal (Sigma #S9811) and 1g Seaplaque low melt agarose (Cambrex #CLNZR006) to 100mL of the LB + 0.1M HEPES pH=6.6 solution in a 150mL screw cap Pyrex bottle with a stirbar. ',4,16),('Eco900_00','procedure','5. Autoclave. ',5,17),('Eco900_00','procedure','6. Carefully place the hot bottle with the molten agarose media solution onto a stirplate. Stir gently for no more than 10 minutes, being sure not to allow the media to cool below 50ºC. ',6,18),('Eco900_00','procedure','7. Add 15mL of the molten agarose solution to a 50mL sterile conical tube in a 42ºC heat block and allow to cool for to 42°C (~1 hour). ',7,19),('Eco900_00','procedure','8. Add 500µL of the starter culture to a microcentrifuge tube (1.7mL). Spin at 13k rpm in a small tabletop centrifuge for 1 min to pellet the cells. Remove the supernatant media. Resuspend the pellet in 500µL fresh (unbuffered) LB media. ',8,20),('Eco900_00','procedure','9. Pellet and resuspend twice more. ',9,21),('Eco900_00','procedure','10. Add Ampicillin, Kanamycin and Chloramphenicol (concentrations as in step 2) to the 42ºC molten agarose. Promptly inoculate the molten agarose with 150µL of the triple washed bacterial suspension. Immediately mix well, but be careful not to introduce bubbles into the media. ',10,22),('Eco900_00','procedure','11. Immediately pour the inoculated media into a 90mm sterile Petri Dish and tilt the dish if necessary to ensure evenness of the media. ',11,23),('Eco900_00','procedure','12. Allow the inoculated agarose solution to solidify at room temperature for 1 hour. ',12,24),('Eco900_00','procedure','13. After the agarose slab has set, cover the top of the Petri Dish with cellophane and poke several slits with a razor blade. ',13,25),('Eco900_00','procedure','14. Place the agarose slab in the incubator, under the image. ',14,26),('Eco900_00','procedure','15. Incubate at 36ºC for 24 hours. ',15,27),('Eco900_00','procedure','16.Remove the plate and place at 4ºC overnight. ',16,28),('Eco900_01','material','Plasmids: pETcoco-1, pDestPBBAC, pDestRmceBAC, pCDNA5ins, pCDNA5/FRT/TO.',1,29),('Eco900_01','procedure','1. Recombinant DNA construct: From a list of 20-bp-long potential TALE binding sites that are orthogonal to gene promoters in the human genome, we selected one that is also orthogonal to mouse gene promoters. The corresponding TALE, TAL118, was assembled using a Golden-Gate cloning scheme (13). The split intein fragments, which have been described previously, were fused to split TALE fragments using polymerase chain reaction and Alw26I Type-IIS restriction enzyme methods. Expression constructs were generated by combining BioBrick subparts using Biobrick assembly. For testing circuits with each part on a separate plasmid, DNA fragments were cloned between NheI and NotI sites of vector pCDNA5ins and reporter fragments between SpeI and NotI sites of pCDNA5/FRT/TO for mammalian expression (Invitrogen). For assembly of circuits on a single DNA construct, an isothermal assembly-based hierarchal cloning scheme was used (Torella et al., submitted). Circuits were assembled on two modified versions of vector pETcoco-1 (Novagen); pDestRmceBAC, which includes two inverted loxP sites around the assembled circuit, and pDestPBBAC, which contains a Blasticidin resistance marker, two inverted terminal repeats and a PiggyBac transposase.',1,30),('Eco900_01','procedure','2. Cell culture:The human osteosarcoma-derived epithelial cell line U-2 OS (ATCC no. HTB-96) was maintained and transfected as previously described (5). TC-1 ES cells (background 129S6/SvEvTac) containing a recombinase-mediated cassette exchange (RMCE) target site were cultured as previously described (23). Transient transfection of ES cells was performed in six-well plates seeded with _750 000 cells using 10 ml of Lipofectamine 2000 reagent (Invitrogen) with 3 mg total DNA per well. A summary of plasmid amounts used for transfections can be found in Supplementary Table S2. For low-copy random integrations using the PiggyBac system, ES cells were transfected using Lipofectamine and selected with 10 mg/ml Blasticidin (Invitrogen) for 7 days starting 48 h post-transfection. ES cell lines with single-copy integration in the beta-globin locus were obtained by RMCE, as previously described (23). In brief, 4_106 ES cells were electroporated with 25 mg of target vector and 15 mg of the Cre Recombinase expression plasmid pIC-Cre. Selection with 3 mM ganciclovir (Roche) was started 48 h post electroporation and continued for 10–12 days. Resistant clones were tested for successful insertion by polymerase chain reaction. Only clones that integrated the target construct in the same orientation were compared.',2,31),('Eco900_01','procedure','3. Microscopy: Microscopy of transiently transfected U-2 OS was performed on live cells 48 h post-transfection. Cells were imaged by a Nikon TE-2000 microscope with a 20_PlanFluor NA=0.5, DIC M/N2 objective and an ORCA-ER charge-coupled device camera. Data collection and processing were performed using Metamorph 7.0 software (Molecular Devices). All images within a given experimental set were collected using the same exposure times, averaged over three frames and underwent identical processing.',3,32),('Eco900_01','procedure','4. Flow cytometry: Approximately 50 000–100 000 live cells were analyzed using an LSRII cell analyzer (BD Biosciences). Cells were trypsinized, pelleted and resuspended in Dulbecco’s phosphate- buffered saline containing 0.1% fetal bovine serum. For transient transfection experiments, output was assayed 48 h post-transfection. To control for variations in transfection efficiency among different samples, we gated for cells that are positive for mCherry (mCh), which is expressed from a constitutive cytomegalovirus (CMV) promoter in all tested constructs. Within these mChpositive cells, we determined the percentage of AmCyan fluorescent protein (CFP)-positive cells (% CFP+ in mCh+). For testing the background activity of the reporter plasmid, it was co-transfected with a mCh-tagged off-target TALE. The offtarget TALE, TAL248, binds to TATACTATCCAATCCAACTT.',4,33),('Eco900_01','equipment','Nikon TE-2000 microscope',1,34),('Eco900_01','equipment',' Metamorph 7.0 software',2,35),('Eco900_01','equipment','LSRII cell analyzer',3,36),('Eco900_02','equipment','standard broad-spectrum glowing lamp (Venture Uni-FORM MH 100W/U/PS R Kr85 M90)',1,37),('Eco900_02','equipment','aluminum foil',2,38),('Eco900_02','equipment','(HP8453) spectrophotometer',3,39),('Eco900_02','equipment','100W Mercury vapor lamp',4,40),('Eco900_02','equipment','EPP2000C Concave Grating spectrometer (Stellarnet, Oldsmar, FL)',5,41),('Eco900_02','material','Luria-Bertrani Media: Ampicillin (25µg /mL), ',1,42),('Eco900_02','material','Chloramphenicol (34µg /mL)',2,43),('Eco900_02','material','Kanamycin (10µg/mL)',3,44),('Eco900_02','material','cyanobacterial medium BG11 (ATCC medium 636)',4,45),('Eco900_02','material','Escherichia coli RU1012 [MC4100 ara+ (OmpC-lacZ) 10-25 envZ::KanR]',5,46),('Eco900_02','material','pPROTet (Clontech)',6,47),('Eco900_02','material','E.coli strain CP919',7,48),('Eco900_02','material','pPRO-HK plasmid',8,49),('Eco900_02','material','RU1012 strains',9,50),('Eco900_02','procedure','Plasmid Construction:The chimeric phytochromes were cloned into the pPROTet (Clontech). This plasmid contains a Chloramphenicol resistance gene and a ColE1 ori, which is compatible with the p15a ori pPL-PCB plasmid. This plasmid also contains the tet promoter, which was used to constitutively drive the expression of the chimeric phytochromes (all of the E. coli strains used do not contain the tetR gene). Prior to cloning the phytochrome fragments, the histidine kinase domain of envZ was ligated into pPROTet producing pPRO-HK. This domain consists of the C-terminal 229 amino acids that have been previously successful in constructing sensor chimeras. The wild-type nucleotide sequence at this cut point encodes the NdeI restriction site. The envZ fragment was amplified from E. coli K12 genomic DNA using primers containing the NdeI and XbaI restriction sites and then cloned using these enzymes into the pPROTet plasmid. The phytochrome fragments were then inserted during a second ligation using the KpnI and NdeI sites.',1,51),('Eco900_02','procedure','Phytochrome Libraries: Using the pPRO-HK plasmid, a number of phytochrome signaling domains were inserted using the KpnI/NdeI restriction sites and tested for activity. First, we ran a CLUSTALW alignment of EnvZ against Cph1. A small library was then produced by varying the number of Cph1 amino acids included (± 9 amino acids), while holding the number of EnvZ amino acids fixed. The most active chimera occurred with one less Cph1 amino acid than predicted from the sequence alignment. The active chimera contains the first 517 amino acids of Cph1 and the last 229 amino acids of EnvZ (cloned into pPRO-HK to form pCPH8). In addition, there is a single I-V mutation at position 136 of the HK EnvZ domain. There were no amino acid substitutions in the Cph1 domain.',2,52),('Eco900_02','procedure','Screening Assay for Light Induction:RU1012 strains transformed with pPL-PCB and a plasmid encoding one of the chimeras were grown overnight (~12 hours) at 37 °C and shaken at 250 rpm in 5 ml LB containing Ampicillin, Kanamycin, and Chloramphenicol. The overnights are then diluted 1:1000 into 2 mL fresh media containing antibiotics in two Falcon 24-well plates. One plate was exposed with a standard broad-spectrum glowing lamp (Venture Uni-FORM MH 100W/U/PS R Kr85 M90) and the other was wrapped in aluminum foil and kept in darkness. Both plates were maintained at 37°C. Each strain was then diluted 1:100 into a new, identical set of 24-well plates and the exposure procedure repeated for another 4 hours. The cells were then collected for analysis of ompC-promoter driven expression of beta-galactosidase. To perform the Miller assay, 2 mL of the final culture were spun at 6000 rpm for 10 minutes in a microcentrifuge and then resuspended in 2mL of Z buffer. The cell density was recorded by measuring the OD600 using a (HP8453) spectrophotometer. Then, 0.5 mL of the resuspended cells were diluted to 1mL in Z buffer and permeabilized by the addition of 100µL of chloroform and 50µL of 0.1% SDS. This mixture was vortexed and allowed to equilibrate at 28 °C for 5 minutes. Then, 0.2 mL of a solution containing 4 mg/ml o-nitrophenyl-beta-galactoside (ONPG, Sigma) was added to the permeabilized cells and incubated at 28°C. The assay was allowed to proceed until sufficient yellow color developed to be seen by eye, the time was recorded and the assay quenched by the addition of 0.5 mL of 1M Na2CO34 with vortexing. The chloroform was removed by centrifuging 1 mL of the quenched reaction at 13,200 rpm for 5 minutes. Finally, the OD420 and OD550 were measured. The Miller units were then calculated using the formula: 1000)*[(OD420-1.75*OD550)] / [(Time of reaction)*(volume of cells)*OD600]. The final OD of the light and dark samples ranged from 0.5 – 0.6.',3,53),('Eco900_02','procedure','Bacterial Photography:E. coli strain CP919 was transformed with pPCB and pCPH8 for the bacterial photography experiments. 50 µl of a 5 ml overnight culture grown to saturation was used to inoculate 50 ml of LB plus 4% Seaplaque Agarose (Cambrex, Pittsburg, PA) with appropriate antibiotics at 40°C. The LB agarose contained 15 mg of the LacZ substrate 3,4-cyclohexenoesculetin-beta-D-galactopyranoside (S-Gal) and 25 mg of ferric ammonium citrate (Sigma Aldrich, St. Louis, MO). The molten cells/media were poured into a flat-surfaced mold, solidified, and grown at 37°C for 12 hours with the image projected onto the slab surface using a 100W Mercury vapor lamp, focusing on the slab surface. The Mercury lamp has power characteristics of .1331 watts/cm2 from 620-680nm (active state of the phytochrome) and 0.0132 watts/cm2 from 715-755nm (inactive state of phytochrome), as determined by a EPP2000C Concave Grating spectrometer (Stellarnet, Oldsmar, FL). For laser repression, a 5mW red laser diode with an output of 0.45 watts/cm2 from 640-680 nm and .0018 watts/cm2 from 715-755 nm when projected across the slab surface.',4,153),('Eco900_03','material',NULL,0,155),('Eco900_03','procedure','',0,156),('Eco900_03','equipment',NULL,0,157);
/*!40000 ALTER TABLE `protocol` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `reaction`
--

DROP TABLE IF EXISTS `reaction`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `reaction` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `reactant1` varchar(60) DEFAULT NULL,
  `reactant2` varchar(60) DEFAULT NULL,
  `product1` varchar(60) NOT NULL,
  `product2` varchar(60) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=7 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `reaction`
--

LOCK TABLES `reaction` WRITE;
/*!40000 ALTER TABLE `reaction` DISABLE KEYS */;
INSERT INTO `reaction` VALUES (1,'S90005','S90006','S90008',NULL),(2,'S90002','S90003','S90006',NULL),(3,'S90004','S90000','S90009',NULL),(4,'S00001','','S00000',NULL),(5,'S00009',NULL,'S00016',NULL),(6,'S00034','S00035','S00037',NULL);
/*!40000 ALTER TABLE `reaction` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `relation`
--

DROP TABLE IF EXISTS `relation`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `relation` (
  `relationcno` varchar(5) NOT NULL,
  `stimulation` varchar(7) NOT NULL,
  `target` varchar(7) NOT NULL,
  `realation` varchar(2) NOT NULL,
  PRIMARY KEY (`relationcno`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `relation`
--

LOCK TABLES `relation` WRITE;
/*!40000 ALTER TABLE `relation` DISABLE KEYS */;
INSERT INTO `relation` VALUES ('r0001','S00006','00021','-'),('r0002','S00008','S00014','-'),('r0003','S00008','00030','-'),('r0004','S00006','00037','-'),('r0005','S00015','S00010','-'),('r0006','S00010','00042','-'),('r0008','S00017','00052','*'),('r0009','S00018','S00019','-'),('r0010','S00020','S00019','-'),('r0011','S00024','S00021','+'),('r0012','S00021','00070','-'),('r0013','S00006','00070','-'),('r0014','S00021','00078','+'),('r0015','S00030','00094','+'),('r0018','S00037','00123','+'),('r0019','S00017','00109','+'),('r0020','S00032','00114','+'),('r1000','S10005','10077','+'),('r1001','S10001','10081','+'),('r1002','S10002','10083','-'),('r2000','S20002','20004','*'),('r2001','S20002','20018','*'),('r2002','S20004','20023','*'),('r2003','S20005','20026','-'),('r2004','S20006','20030','*'),('r2005','S20007','20040','-'),('r2006','S20007','20039','-'),('r2007','S20004','20050','*'),('r2008','S20009','20051','*'),('r2009','S20019','20078','-'),('r2010','S20021','20082','-'),('r2011','S20020','20079','-'),('r2012','S20005','20109','+'),('r2013','S20005','20111','+'),('r2014','S20005','20113','+'),('r2015','S20021','20109','-'),('r2016','S20021','20111','-'),('r2017','S20021','20113','-'),('r2018','S20028','S20025','-'),('r2019','S20024','S20028','-'),('r2020','S20022','S20024','+'),('r2021','S20023','S20022','-'),('r2022','S20022','S20023','-'),('r2023','S20027','S20026','+'),('r2024','S20022','S20027','-'),('r2025','S20023','S20027','+'),('r2026','S20029','20120','-'),('r2027','S20030','20120','-'),('r2028','S20031','20120','-'),('r4000','S40001','40004','+'),('r4001','S40001','40008','+'),('r4002','S40002','40008','-'),('r4003','S40003','40020','+'),('r4004','S40004','40016','-'),('r4005','S40005','40020','-'),('r4006','S40007','40035','+'),('r4007','S40011','40049','+'),('r9006','S90007','S90008','-'),('r9007','S90009','90008','+'),('r9008','S90001','90008','-'),('r9009','S90008','90000','+');
/*!40000 ALTER TABLE `relation` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `substance`
--

DROP TABLE IF EXISTS `substance`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `substance` (
  `substanceno` varchar(6) NOT NULL,
  `substancename` varchar(90) NOT NULL,
  `existance` varchar(1) NOT NULL,
  PRIMARY KEY (`substanceno`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `substance`
--

LOCK TABLES `substance` WRITE;
/*!40000 ALTER TABLE `substance` DISABLE KEYS */;
INSERT INTO `substance` VALUES ('S00000','O-RBS','-'),('S00001','O-16S rRNA','-'),('S00002','FT','-'),('S00003','Alpha-acetolactate synthase protein','-'),('S00004','Ketolacid reductoisomerase,NAD(P)-binding','-'),('S00005','Dihydroxyacid dehydratase','-'),('S00006','TetR ','-'),('S00007','Alpha-ketoisovalerate decarboxylase','-'),('S00008','FadR','-'),('S00009','LuxI','-'),('S00010','LacI','-'),('S00011','Glp-1','-'),('S00012','CPP','-'),('S00013','LuxR','-'),('S00014','Fatty Acyl-CoA','-'),('S00015','lactonase','-'),('S00016','AHL','-'),('S00017','LuxR/AHL cpmlex','-'),('S00018','Clts','-'),('S00019','Pcl','-'),('S00020','temperature 25 centigrade','-'),('S00021','Light sensor','-'),('S00022','GFP','-'),('S00023','tetO','-'),('S00024','Light','-'),('S00025','P(3HB) synthase','-'),('S00026','phaA','-'),('S00027','phaB','-'),('S00028','Ag43','-'),('S00029','eCFP','-'),('S00030','L-arabinose','-'),('S00031','lysostaphin protein','-'),('S00032','marR','-'),('S00033','lysis E7','-'),('S00034','agrC','-'),('S00035','agrA','-'),('S00036','yddV','-'),('S00037','agrA~P','-'),('S10000','blue light sensor protein','+'),('S10001','response regulator FixJ','+'),('S10002','cl repressor protein','-'),('S10003','480nm light(blue)','-'),('S10004','Cytolysin A pore forming protein','+'),('S10005','IPTG','+'),('S20000','AHL','-'),('S20001','LuxR protein','-'),('S20002','AHL-LuxR complex ','-'),('S20003','3OC6HSL','-'),('S20004','NahR','-'),('S20005','AraC','-'),('S20006','T7 polymerase','-'),('S20007','CI protein','-'),('S20008','CI434 protein','-'),('S20009','PhiR73 delta/T3pol protein','-'),('S20010','LuxP-AI-2 complex','-'),('S20011','LuxQ kinase','-'),('S20012','LuxU protein','-'),('S20013','LuxP protein','-'),('S20014','LuxO protein','-'),('S20015','transcription factor sigma 54','-'),('S20016','mRNA encoding the transcriptional activator  LuxR4','-'),('S20017','Lac O2 protein','-'),('S20018','CIS Mrna','-'),('S20019','TRANS mRNA','-'),('S20020','IPTG','-'),('S20021','LacI protein','-'),('S20022','cI','-'),('S20023','cro','-'),('S20024','cII','-'),('S20025','cIII','-'),('S20026','UV','-'),('S20027','RecA','-'),('S20028','Hfl','-'),('S20029','dioxin','-'),('S20030','Ahr','-'),('S20031','ARNT','-'),('S20032','Leucine dehydrogenase','-'),('S20033','Leucine','-'),('S20034','isovaleric acid','-'),('S40000','GFP protein','-'),('S40001','luxR protein','-'),('S40002','CI434 protein','-'),('S40003','T7 RNAP','-'),('S40004','tetR protein','-'),('S40005','cI protein','-'),('S40006','T7 RNA polymeraze','-'),('S40007','LacI protein','-'),('S40008','SupD protein','-'),('S40009','lignin peroxidase ','-'),('S40010','GFP Light protein','-'),('S40011','presure about 30 MPa','-'),('S90000','AHL','-'),('S90001','cI transcriptional repressor','-'),('S90002',' heme oxygenase 1','-'),('S90003','phycocyanobilin:ferredoxin oxidoreductase','-'),('S90004','LuxR transcription factor','-'),('S90005','cph8 chimeric sensor kinase','-'),('S90006','PCB','-'),('S90007','red light','-'),('S90008','PCB/cph8 complex','-'),('S90009','AHL/LuxR complex','-'),('S90010','beta-galactosidase','-');
/*!40000 ALTER TABLE `substance` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2013-09-21 19:20:04
